go.mod

```diff
@@ -62,6 +62,7 @@ require (
 	github.com/hashicorp/cap v0.2.1-0.20220727210936-60cd1534e220
 	github.com/hashicorp/consul-template v0.29.5
 	github.com/hashicorp/consul/api v1.15.2
+	github.com/hashicorp/consul/sdk v0.11.0
```

Comment:
I might be missing it, but I don't see this dependency being used. Should this be removed?

Reply:
💯 I neglected to `go mod tidy` thanks for catching this.